### DISCUSSION TITLE
Τελική παρουσίαση προόδου σε άλλες ημέρες ώρες

### DISCUSSION THREAD

**Discussion by perikarananou** [courses-ionio/sw#1610] [Q&A] [answered]
Σε περίπτωση που κάποιοι δεν καταφέρουν να βρίσκονται στην Κέρκυρα για την τελική παρουσίαση όπου  σύμφωνα με οδηγίες θα είναι διαθέσιμη για πλήρη βαθμολόγηση μόνο για την 10η εβδομάδα (8 και 9 Μαίου), αυτόματα πάμε με μειωμένο βαθμό?

**Comment by KonstantinosTourtsakis** [comment, marked as answer]
https://epidrome.github.io/teaching/progress/
Η ενδιάμεση και η τελική παρουσίαση είναι υποχρεωτικές, οπότε θα πρέπει να πραγματοποιηθούν για να περάσεις το μάθημα. 
> Η προφορική παρουσίαση (ενδιάμεσης και τελικής) προόδου είναι υποχρεωτική και πρέπει να γίνουν τουλάχιστον δύο εμπρόθεσμα ραντεβού.

Όπως ανέφερες θα υπάρξει μείωση του βαθμού εάν η παρουσίαση γίνει μετά από την προβλεπόμενη εβδομάδα.
> Για όσους παρουσιάσουν αργότερα ο βαθμός θα είναι μειωμένος γιατί έχουν περισσότερο χρόνο για την προετοιμασία και μετά από δύο εβδομάδες (7η και 12η εβδομάδα) βαθμολογούνται με άριστα την βάση. Στην εξεταστική περίοδο δεν γίνονται παρουσιάσεις, γιατί γίνεται η ομότιμη αξιολόγηση και ο έλεγχος-διόρθωση των αιτημάτων ενσωμάτωσης.

Ο βαθμός μειώνεται αν δεν έχεις παρουσιάσει μέχρι και την 10η εβδομάδα και από την 12η εβδομάδα (7η για την ενδιάμεση) και μετά παίρνει άριστα το 5. Η ενδιάμεση παρουσίαση αφορά τα παραδοτέα 1-5 και η τελική αφορά τα παραδοτέα 6-9.

Αν έχεις πραγματοποιήσει εγκαίρως την ενδιάμεση παρουσίαση λογικά η τελική σε επηρεάζει μόνο για τα παραδοτέα που της αντιστοιχεί.

**Reply by perikarananou** [reply]
Σε ευχαριστώ πολύ! Αρα από την 10η μέχρι την 12η εβδομάδα δεν υπάρχει καμία μείωση βαθμού. Δηλαδή η τελική παρουσίαση είναι από 8/5 μέχρι και 23/5;

**Reply by KonstantinosTourtsakis** [reply]
Όχι, όπως το καταλαβαίνω αν παρουσιάσεις την 11η ή την 12η εβδομάδα θα υπάρξει μια μικρή μείωση του βαθμού (1-3 μονάδες) ενώ εάν τελικά η παρουσίαση γίνει μετά την 12η εβδομάδα τότε ξεκινάς με άριστα την βάση.

Σύμφωνα με την [τελευταία ανακοίνωση](https://github.com/courses-ionio/sw/discussions/1607) μπορείς να αποφύγεις βαθμολογική επίπτωση εάν παρουσιάσεις μέχρι τις 9 Μαίου.
> Τέλος σας θυμίζουμε η τελική παρουσίαση προόδου θα είναι διαθέσιμη για πλήρη βαθμολόγηση μόνο για την 10η εβδομάδα, δλδ 8 και 9 Μαίου.

**Reply by epidrome** [reply]
επίσης μπορείτε να παρουσιάσετε έγκαιρα στον συνάδελφο αν δεν σας βολεύουν οι μέρες-ώρες εδώ #1408
